content/posts/2021-features.md

```diff
@@ -0,0 +1,132 @@
+---
+title: "2021 for Packit"
+date: 2022-01-12T09:23:38+01:00
+draft: false
+weight: 65
+---
+
+# Packit project in 2021
+
+The previous year 2021 wasn't interesting only because of the increased usage of Packit
+(you can see more in [the previous post](../2021-in-numbers)).
+The whole Packit team made a lot of improvements during the year.
+Some small, some really big. So, let's take a look at the most important ones:
+
+## Dashboard
+
+The idea of having a dashboard for Packit service started as a Google Summer of Code 2020 project
+to provide a basic view of our service.
```

Comment:
Please give credit to @IceWreck, and maybe link one of his [progress reports](https://communityblog.fedoraproject.org/gsoc-progress-report-dashboard-for-packit-july-1-aug-16-2020/).

Reply:
Very good point! Something like this?

Reply:
Yes, thanks! :+1: